lib/web_ui/test/paragraph_test.dart

```diff
@@ -426,6 +426,209 @@ void main() async {
     expect(paragraph.getBoxesForRange(0, 0), isEmpty);
   });
 
+  testEachMeasurement('getBoxesForRange multi-line', () {
```

Comment:
Is it worth adding a test for a `ParagraphStyle` with `maxLines` such that the `start`/`end` range fall outside the max line bounds?

Reply:
That's a good corner case to test. I'd like to merge this PR while the tree is green though. So I'll add the `maxLines` test in a future PR.